dashboard/app/models/sections/section.rb

```diff
@@ -224,6 +224,23 @@ def update_student_sharing(sharing_disabled)
     end
   end
 
+  after_save :ensure_owner_is_active_instructor
+  def ensure_owner_is_active_instructor
+    return if user.blank?
```

Comment:
Should this be an logged error state?

Reply:
I don't think so. Earlier in the file we have
```ruby
  validates_presence_of :user, unless: -> {deleted?}
```
which implies it's normal and expected for the user to be `blank` if the section is soft-deleted. The presence of the user is already validated there anyway when the section isn't deleted, so no need for a duplicate error here.